src/main/scala/org/dberg/hubot/listeners/TestListener3.scala

```diff
@@ -0,0 +1,21 @@
+package org.dberg.hubot.listeners
+
+import org.dberg.hubot.Hubot
+import org.dberg.hubot.listeners.Listener.CallbackSuccess
+import org.dberg.hubot.models._
+
+class TestListener3(hubot: Hubot) extends Listener(hubot) {
+
+  val callback: Callback = {
```

Comment:
I think I see.  I guess the only challenge there is no real way to know if the listener ran successfully if it didnt throw an exception but I think thats fine for now

  if (callback.isDefinedAt(updated)) {
        try {
          callback(updated)   //have to assume this worked 
          CallbackSuccess
        } catch {
          case NonFatal(e) => CallbackFailure(e)
        }